.github/CODEOWNERS

```diff
@@ -679,5 +679,5 @@
 /scripts/install_third_party_libs.py @vojtechjelinek
 /.github/ @DubeySandeep
 /.github/CODEOWNERS @DubeySandeep
-/.github/stale.yml @ankita240796
+/.github/stale.yml @vojtechjelinek
```

Comment:
Not really sure why this was assigned to me, but I guess it is okay since this is rarely modified.